apps/andi/assets/css/metadata_form.scss

```diff
@@ -1,6 +1,7 @@
 .metadata-form-edit-section {
   grid-template-areas:
     "title last-updated"
+    "data-name ."
```

Comment:
do you want that space dot on the end?

Reply:
yeah, it's the goofy grid css way of adding a blank (or stretching it?) cell

Reply:
yeah I think that is how you denote an empty space when defining a grid elsewhere in andi, but the styling is not currently lining up so I will look into it some more